web/js/duplicates.js

```diff
@@ -13,9 +13,15 @@ $(function() {
         // hooked onto it, once the ajax request as completed.
         var dfd = $.Deferred();
 
+        var category = $('select[name="category"]').val();
+        if (category === '-- Pick a category --') {
```

Comment:
My Perl is a little rusty, but I’m assuming the `_('-- Pick a category --')` in `DB/Result/Problem.pl` and `Controller/Report/New.pl` mean this string can be translated?

If so, should we somehow check for the (potentially translated) string here, in the JavaScript, rather than just the English? Could do it with a `data-` attribute containing the "default" value to check against, perhaps?

Reply:
Pick a category is already buggy (see it's not translated anywhere in fixmystreet.js) so can be fixed separately (especially as this isn't generically used code at present).